plugin/my_asset_manager/MyAssetManagerInterface.py

```diff
@@ -121,6 +120,95 @@ def isEntityReferenceString(self, someString, hostSession):
         # info()
         return someString.startswith(self.__reference_prefix)
 
+    def entityTraits(
+        self,
+        entityReferences,
+        entityTraitsAccess,
+        context,
+        _hostSession,
+        successCallback,
+        errorCallback,
+    ):
+        # This function is used by the host to retrieve the trait sets
+        # for specific entities. The behaviour of this function differs
+        # per access mode. `kRead` is a request for an exhaustive trait
+        # set for an entity according to this manager, whilst `kWrite`
+        # is a request for the minimal trait set required to publish to
+        # that entity. As this manager example is read-only, we will
+        # simply reject any `kWrite` requests.
+
+        # For the purposes of this template, we use this fake map
+        # of LocatableContent paths to serve as our "database".
+        # Replace this with querying your backend systems.
+        managed_filesystem_locations = {
```

Comment:
[fix] This isn't used anywhere

Reply:
Did your idea in the other comment : https://github.com/OpenAssetIO/Template-OpenAssetIO-Manager-Python/pull/19/commits/d14e9d3bae80e53b33cb15a97f6687046fe19245

Reply:
Lovely!